src/oops/designPatterns/prototype/Registry.java

```diff
@@ -11,9 +11,11 @@
 //Singleton
 public class Registry {
 
-	private Map<String, Item> items = new HashMap<String, Item>();
+	private static Map<String, Item> items = new HashMap<String, Item>();
```

Comment:
Why do we have to make it static? Why not it should be only a private attribute?

Reply:
yeah ..